desktop/apps/auction/components/ArtworkBrowser/ArtworkBrowser.js

```diff
@@ -1,9 +1,9 @@
-import ArtworkDisplay from './main/ArtworkDisplay'
-import HeaderDesktop from './header/HeaderDesktop'
-import HeaderMobile from './header/HeaderMobile'
+import ArtworkDisplay from './Main/ArtworkDisplay'
+import HeaderDesktop from './Header/HeaderDesktop'
+import HeaderMobile from './Header/HeaderMobile'
```

Comment:
A lot of these renamings are to bring our react bits in line with Reaction / Emission